docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -79,4 +79,36 @@ include::{sourcedir}/clustered/client/src/test/java/org/ehcache/clustered/client
     Caches can also be configured with shared resource pools where multiple caches can share the resources from the same resource pool.
     This can be achieved using `.shared(String)`, passing the name of the resource pool. The resource pool name can be used by other caches to share the resource pool.
 <8> Creates fully initialized cache manager with the clustered cache(*clustered-cache*).
-<9> Close the cache manager.
\ No newline at end of file
+<9> Close the cache manager.
+
+== Clustered caches
+
+IMPORTANT: Content needed here!
+
+=== Specifying consistency level
+
+Ehcache offers two level of consistency:
+
+Eventual::
+In _eventual_ consistency, the clustering solution will make sure that when a mutative operation returns, invalidation for the corresponding entries found on other clients has been initiated.
+This is the default level if not configured explicitly.
+Strong::
+In _strong_ consistency, the clustering solution will make sure that when a mutative operation returns, the corresponding entries found on other clients have been invalidated.
```

Comment:
Invalidation of keys on clients seems to be an internal thing, I think we should document strong vs eventual by explaining the visibility of change available to other clients.